spec/browser_spec.rb

```diff
@@ -87,11 +87,19 @@ module Ferrum
       expect(browser.viewport_size).to eq([200, 400])
     end
 
-    it "allows the viewport to be resized by fullscreen" do
+    it "allows the viewport to be resized to fullscreen and back" do
```

Comment:
I do believe it should be in a dedicated test, like: `resizes to "normal" from "fullscreen" window state`

Reply:
I don't remember for sure but maybe such test would need to switch from windowed to fullscreen first? Then it would repeat "windowed to fullscreen".

Reply:
it's okay to "repeat" coz, we have 2 different cases:
1 - resize with fullscreen mode -> viewport_size depends on document sizes
2 - resize from fullscreen mode to normal -> viewport_size depends on specified arguments